{{ cookiecutter.repo_name }}/{{ cookiecutter.repo_name }}/structure.py

```diff
@@ -0,0 +1,33 @@
+"""Define global project (file-)structure here.
+
+This module is used to define all filepaths and/or keys
+used in the project. the __factory function is used to build
+these paths or path templates on settings or environment variables.
+
+Naming convention:
+all caps for paths and lower case names for path templates.
+"""
+
+from os import path
+from drtools.conf import settings
+
+PATH_TEMPLATES = {}
```

Comment:
I'd like to have constant paths separated from templates.

Reply:
I agree with @michcio1234. Seperate dictionaries for templates and constant paths will look cleaner, so +1 for the PATHS dictionary.